tox.ini

```diff
@@ -9,7 +9,6 @@ ignore=D001
 [pycodestyle]
 exclude = .git,.tox
 max-line-length = 120
-max-doc-length = 79
```

Comment:
Do you want to remove the check altogether and not go for `max-doc-length = 120`?

Reply:
I was following edx-django-utils configuration. But then I found this: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization

> Please use 79 chars as a limit for docstring lines though, so that the text remains readable.

I think I'm gonna close this issue. Thanks for the review though!

Reply:
I fixed it! @MoisesGSalas